### DISCUSSION TITLE
Pets

### DISCUSSION THREAD

**Discussion by XelaAl** [NAIST-SE/AttractingContributors#628] [General]
I've recently adopted a dog and now I'm in search of any useful information on how to take care of her. What products do you use to protect your pets from fleas and ticks? Where do you buy them? Any advice is appreciated.

**Comment by hernessteve** [comment]
I like Frontline most of all. It is not to chemical and worth the money

**Comment by ellielo** [comment]
My congratulations! Adopting a dog is a great decision. But I agree hat it is essential to take care of your pet in a proper way. As for flea products, I order Advantage Multi for my dog  on the [Pet express](https://www.worldpetexpress.net/ ) site. It works just fine and we haven't faced any issues with fleas and ticks.
